src/WebUI/Components/Common.scss

```diff
@@ -13,6 +13,7 @@
 
 .list-col-content {
     align-items: center;
+    line-height: 30px;
```

Comment:
i remember this not having the desired effect on VSTS. you can check later after merging your PR